idd/Energy+.idd.in

```diff
@@ -482,11 +482,12 @@ Building,
        \key Urban
        \default Suburbs
   N2 , \field Loads Convergence Tolerance Value
-       \note Loads Convergence Tolerance Value is a fraction of load
+       \note Loads Convergence Tolerance Value is a change in load from one warmup day to the next
        \type real
        \minimum> 0.0
        \maximum .5
        \default .04
+       \units W
```

Comment:
So the code was already interpreting this input as W, but it was described as a fraction? If so, it appears that the E+ documentation needs to be updated to reflect this.

Reply:
Yeah if this was described as a fraction in the docs then they need to be changed.